engine/src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -92,20 +92,21 @@ object WorkflowManagerActor {
  * WorkflowOutputs: Returns a `Future[Option[binding.WorkflowOutputs]]` aka `Future[Option[Map[String, WdlValue]]`
  *
  */
-class WorkflowManagerActor(config: Config)
+class WorkflowManagerActor(isServerMode: Boolean, config: Config = WorkflowManagerActor.defaultConfig)
   extends LoggingFSM[WorkflowManagerState, WorkflowManagerData] with CromwellActor {
 
-  def this() = this(WorkflowManagerActor.defaultConfig)
   implicit val actorSystem = context.system
 
   private val logger = Logging(context.system, this)
   private val tag = "WorkflowManagerActor"
 
   private val donePromise = Promise[Unit]()
 
+  val isRestartable = config.getConfig("backend").getBooleanOr("workflow-restart", default = true)
```

Comment:
So when I try to run 0.19_hotfix in server or run mode without the backend stanza, I get an error. 
`Exception in thread "main" com.typesafe.config.ConfigException$Missing: No configuration setting found for key 'backend'
	at com.typesafe.config.impl.SimpleConfig.findKeyOrNull(SimpleConfig.java:152)
	at com.typesafe.config.impl.SimpleConfig.findOrNull(SimpleConfig.java:170)
	at com.typesafe.config.impl.SimpleConfig.find(SimpleConfig.java:184)
	at com.typesafe.config.impl.SimpleConfig.find(SimpleConfig.java:189)
	at com.typesafe.config.impl.SimpleConfig.getObject(SimpleConfig.java:258)
	at com.typesafe.config.impl.SimpleConfig.getConfig(SimpleConfig.java:264)
	at com.typesafe.config.impl.SimpleConfig.getConfig(SimpleConfig.java:37)
	at cromwell.server.WorkflowManagerSystem$class.allowedBackends(WorkflowManagerSystem.scala:24)
	at cromwell.Main$$anon$1.allowedBackends(Main.scala:97)
	at cromwell.server.WorkflowManagerSystem$class.$init$(WorkflowManagerSystem.scala:27)
	at cromwell.Main$$anon$1.<init>(Main.scala:97)
	at cromwell.Main.<init>(Main.scala:97)
	at cromwell.Main$.delayedEndpoint$cromwell$Main$1(Main.scala:49)
	at cromwell.Main$delayedInit$body.apply(Main.scala:31)
	at scala.Function0$class.apply$mcV$sp(Function0.scala:34)
	at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:12)
	at scala.App$$anonfun$main$1.apply(App.scala:76)
	at scala.App$$anonfun$main$1.apply(App.scala:76)
	at scala.collection.immutable.List.foreach(List.scala:381)
	at scala.collection.generic.TraversableForwarder$class.foreach(TraversableForwarder.scala:35)
	at scala.App$class.main(App.scala:76)
	at cromwell.Main$.main(Main.scala:31)
	at cromwell.Main.main(Main.scala)`

Based on that I'm assuming it's not skippable? 


Otherwise I've made all the changes that I thought were necessary. If you have time for a final check up on this, I'd love to get it merged prior to the long weekend :) Thanks!